ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -111,9 +129,9 @@
                 (every? #(not= native-id (:native-id %)) active-duplicate-queries))
        (errors/throw-service-error
         :conflict
-        (format (str "The subscriber-id [%s] has already subscribed to the "
-                     "collection with concept-id [%s] using the query [%s]. "
-                     "Subscribers must use unique queries for each Collection.")
+        (format "The subscriber-id [%s] has already subscribed to the "
+                "collection with concept-id [%s] using the query [%s]. "
+                "Subscribers must use unique queries for each Collection."
                 subscriber-id collection-id  normalized-query)))))
```

Comment:
format needs to take a single string as the first argument with the replacements passed in.

Reply:
Thank Jay, should be formatted properly now.